docs/development/database_migrations.rst

```diff
@@ -74,3 +80,80 @@ command to see an output of the SQL that will be generated.
 .. code-block:: none
 
     alembic upgrade head --sql
+
+Testing Migrations
+~~~~~~~~~~~~~~~~~~
+
+The test suite already comes with a test runner (``test_alembic.py``) hat run a series of checks to
+ensure migration's upgrade and downgrade commands are idempotent and don't break the database. The
+test runner uses dynamic module import to iterate through all the migrations. You will need to
+create a python module in the ``tests/migrations/`` directory. You module **MUST** be named
+``migration_<revision identifier>.py``. For example, if your revision is named
+``15ac9509fc68_init.py``, your test module will be named ``migration_15ac9509fc68.py``.
+Example modules for the first two revisions are shown below.
+
+.. code-block:: none
+
+    tests/migrations/
+    ├── __init__.py
+    ├── migration_15ac9509fc68.py
+    └── migration_faac8092c123.py
+
+
+Your module **MUST** contain the classes with the following classes with the following attributes.
```

Comment:
Nit: `the classes with the following classes with the following attributes` -> `the following classes with the following attributes`